controller/tracepipeline/config_builder.go

```diff
@@ -41,9 +41,13 @@ type ExporterConfig struct {
 	Logging  LoggingExporterConfig `yaml:"logging,omitempty"`
 }
 
+type OTLP struct {
```

Comment:
Let's call it `OTLPReceiverConfig` since we already have an `OTLPExporterConfig`